### DISCUSSION TITLE
SSH

### DISCUSSION THREAD

**Discussion by dealerwrx** [WinMerge/winmerge#1888] [Ideas]
How about a comparison on folders that are reached through SSH (thus allowing file compare between windows and linux systems. I would need to allow username@password combinations to login to linux systems.

**Comment by spurgeonbj** [comment]
This would be an awesome leap... 

Considering implementation of SSH Tunnels in open source stuff like VSCode, guess this shouldnt be very difficult...

If some one can orient me, i can try my hand in developing a SSH mode plugin...

**Comment by lededev** [comment]
Use `SSHFS-Win` or `rclone mount` you can mount ssh remote dir to local dir on Windows OS.
